clouddriver-sql/src/main/resources/db/changelog/20201110-field-type-postgres.yml

```diff
@@ -0,0 +1,105 @@
+databaseChangeLog:
+  - changeSet:
+      preConditions:
+        onFail: CONTINUE
+        dbms:
+          type: postgresql
+      id: change-field-types-postgresql
+      author: ncknt
+      changes:
+        - modifyDataType:
+            columnName: id
+            newDataType: varchar(36)
+            tableName: tasks
+        - modifyDataType:
+            columnName: id
+            newDataType: varchar(36)
+            tableName: task_states
+        - modifyDataType:
+            columnName: task_id
+            newDataType: varchar(36)
+            tableName: task_states
+        - modifyDataType:
+            columnName: id
+            newDataType: varchar(36)
+            tableName: task_results
+        - modifyDataType:
+            columnName: task_id
+            newDataType: varchar(36)
+            tableName: task_results
+#        - modifyDataType:
+#            columnName: body
+#            newDataType: text
+#            tableName: task_results
```

Comment:
Is this entry because of tests during development? If it should not be applied it would be better to be removed.